octoprint_mrbeam/templates/laserhead_changed.jinja2

```diff
@@ -1,19 +1,87 @@
 <div id="laserhead_changed" class="modal hide fade" data-backdrop="static">
     <div class="modal-header">
-        <a href="#" class="close" data-dismiss="modal" aria-hidden="true">&times;</a>
-        <h3>{{ _('A new Laserhead has been detected') }}</h3>
+        <h3>{{ _("A new laser head has been detected!") }}</h3>
     </div>
-    <div class="modal-body">
+    {# Not supported laserhead  - START #}
+    <div class="modal-body" data-bind="visible: !laserhead_model_supported">
         <div>
-            <img loading="lazy" src="/plugin/mrbeam/static/img/welcome_dialog/start.png">
-            <h4> {{ _("Your new Mr Beam dreamcut [S] Laserhead has been successfully installed.") }}</h4>
-            <p> {{ _("With this Laserhead you can work now even faster.") }}</p>
-            <p> {{ _("This means that we also update the default material settings to the new laserhead as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
-            <p> {{ _("Read about the changes you can expect with the new and improved MrBeam II dreamcut [S] Laserhead.") }} <a target="_blank" href="{{ _("https://support.mr-beam.org/en/support/solutions/articles/43000640531") }}"> » {{ _("Read more") }} </a></p>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("We are verv sorry but the installed Laser head will not work with vour device.") }}</h4>
+            <p> {{ _("Every device older than the dreamcut row will not support the new and advanced Laser head properly. There are just too many improvements that are missing in the earlier states of our model rows like Mr Beam and Mr Beam II. Please contact our support tor further steps.") }}</p>
         </div>
     </div>
+    {# Not supported laserhead  - END #}
+
+
+    {# Steps for laserhead S  - START #}
+    <div class="modal-body" data-bind="visible: show_step_for_laserhead_s(1)">
+        <div>
+            <img loading="lazy" src="/plugin/mrbeam/static/img/Welcome.png">
+            <h4> {{ _("Your Mr Beam dreamcut [S] Laser head has been successfully installed.") }}</h4>
+            <p> {{ _("With this Laser head you can work now even faster.") }}</p>
+            <p> {{ _("This means that we also update the default material settings to the new laser head as long it is installed to provide you the best possible support. However, we do not update your custom materials, this will need your personal testing and finetuning.") }}</p>
```

Comment:
```suggestion
            <p> {{ _("This means that we also adjust the default material settings to the new laser head as long as it is installed, to provide the best possible support. However, we do not update your custom materials, this will need your personal testing and fine tuning.") }}</p>
```